Classes/CocoaLumberjack.swift

```diff
@@ -1,6 +1,6 @@
 // Software License Agreement (BSD License)
 //
-// Copyright (c) 2014, Deusty, LLC
+// Copyright (c) 2014-2015, Deusty, LLC
```

Comment:
As for updating the copyright, I wonder if "Deusty, LLC" is still pertinent now that it is part of a separate open source organization. @robbiehanson ?